### ISSUE TITLE
Tags in dark mode have low contrast

### ISSUE THREAD

**Issue by delroth** [lukeorth/poison#91]
Screenshot from the demo website:
![out](https://github.com/lukeorth/poison/assets/202798/a155b8f9-0b08-4c34-9fb3-52bc3c6d9cca)

This is below the recommended contrast for website text and is flagged (correctly, imo) as an accessibility issue. In general the colors there are defined weirdly in my opinion:

- Foreground color is hardcoded to white in css/poison.css.
- Background color uses --list-color, which is used as a text foreground color (that's the one used for list elements in posts).

In light mode this works ok because --list-color uses a dark foreground color (as expected from light mode), and white has good contrast with that. But in dark mode this ends up with white on grey instead.

I don't have a strong preference on how to fix this, personally on my website I've overridden the tags to use code background/foreground colors instead and I think it looks nice :p

**Comment by lukeorth** [comment]
I agree, this is an accessibility issue.  Using the code background/foreground colors sounds like a good fix.  I'll play around with it a bit and see what I can find. :slightly_smiling_face:
